src/10-deep-strong-coupling.md

```diff
@@ -0,0 +1,1132 @@
+
+```python
+projection_operators = [basis(len(nm_list), i).proj() for i in range(len(nm_list))]
+```
+
+We'll also extract expectation values for some of the Hamiltonian operators.
+
+```python
+result = sesolve(H, psi0, times, e_ops = projection_operators + [number, U*interaction, omega*bosons])
+
+probs = result.expect[:-3]      # Remove the last 3 expectations to get the state probabilities
+expected = result.expect[-3:]   # The last 3 expectations are the number, interaction and field energies
+```
+
+```python
+plot_sim(times/(2*np.pi/omega), probs ,labels=ket_labels)
+plt.title(f" {H_latex}   \n $\Delta E={DeltaE}$, $\omega={omega}$, $U={U}$, $\Psi_0=|0,->$     (Fig. 1)");
+plt.xlabel("Time ($2\pi/\omega$)");
+```
+
+Fig. 1 shows us that our picture is correct. The vacuum state "pulls" the other state into almost 100% occupation probability on a timescale associated with the [Rabi frequency](https://en.wikipedia.org/wiki/Vacuum_Rabi_oscillation) $\Omega = \sqrt{4U^2 + \delta^2}$ where $\delta$ is often called the "detuning" which is just the difference in energies of the two states. 
+
+```python
+rabi_omega = np.sqrt(4*U**2 + 1**2)
+```
+
+```python
+rapi_period = 1/rabi_omega
+print(rapi_period)
+```
+
+How are the results of Fig. 1 possible? Where has the energy come from to create a boson from nothing? Recall that the TLS has zero energy so it's not from there!
+
+The only place it can come from is the interaction energy. Let's take a look at the expectation of the boson and interaction energy to resolve this.
+
+```python
+plt.figure(figsize=(10,8))
+plt.plot(times/(2*np.pi/omega), expected[2], label="$\hbar\omega(a^{{\dagger}}a +1/2)$ - bosons")
+plt.plot(times/(2*np.pi/omega), expected[1], label="$U( a^{{\dagger}} + a )\sigma_x$ - interaction")
+plt.ylabel("Expected energy")
+plt.xlabel("Time ($2\pi/\omega$)")
+plt.title(f" {H_latex}   \n $\Delta E={DeltaE}$, $\omega={omega}$, $U={U}$, $\Psi_0=|0,->$     (Fig. 2)");
+plt.legend(loc='right');
+plt.show();
+```
+
+Fig. 2 shows is that the boson energy is being "borrowed" from interaction energy which can take on a form of energy debt. We actually saw hints of this way back in tutorial 4 (Fig. 13), but now that the coupling is so much stronger we can see the effect more clearly.
+
+
+### Arbitrary number of coupled states
+
+
+What happens when we remove the artificial restriction of having only two states? How does our picture change?
+
+A significant change is that the coupling between the states is not constant - it depends on how many field quanta $n$ we have. This is because of how the field operators work:
+
+$$
+\begin{aligned}
+a^{\dagger} |n,\pm\rangle &= \sqrt{n+1}|n+1,\pm\rangle \\
+a |n,\pm\rangle &= \sqrt{n}|n-1,\pm\rangle \\
+a^{\dagger}a |n,\pm\rangle &= n|n,\pm\rangle
+\end{aligned}
+$$
+
+Far from being constant, the coupling scales as $\sim 2U\sqrt{n}$. Our picture is therefore going to look something like:
+
+![](./img/pendulums-dsc-02.png)
```

Comment:
Again, the picture confuses me: doesn't the + and - denote excited state and ground state of the TLS? Doesn't the picture then depict + - - - -?

Reply:
I understand what you're seeing.

But, the pendulum picture needs to be seen as every state has its own energy and it's own pendulum.

Check out [Fig 6 in Tutorial 9](https://nbviewer.org/github/project-ida/two-state-quantum-systems/blob/master/09-destructive-interference.ipynb). Below that figure we map the pendulums to the 4 states involved in exctiation transfer.

Reply:
Nice idea. I've added the link in the "From classical to quantum"